<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>831</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Record</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Replay</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_replay">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>OpenFile</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_ais_list">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>521</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_source_list">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>60</y>
        <width>221</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_time">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>time</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
